config/saucelabs.yml

```diff
@@ -0,0 +1,2 @@
+username: aballantine
+authkey: c09c4ae6-c68f-419d-b09a-922404c5bb23
```

Comment:
Should this be in a public repo ?  Does  it not mean someone else could use our account ?